packages/web-app-files/tests/unit/mixins/actions/restore.spec.js

```diff
@@ -79,8 +102,21 @@ function getWrapper({ invalidLocation = false, resolveClearTrashBin: resolveRest
       },
       $gettext: jest.fn(),
       $gettextInterpolate: jest.fn(),
+      createModal: jest.fn(),
+      $clientService: {
+        webdav: {
+          listFiles: jest.fn().mockImplementation(() => {
+            return []
+          })
+        }
+      },
       $client: {
         ...sdkMock,
+        files: {
+          list: jest.fn().mockImplementation(() => {
+            return Promise.resolve([])
+          })
+        },
```

Comment:
This is probably not needed anymore after you ported your code to `webdav.listFiles`